custom_components/tuya_local/devices/airquality6in1.yaml

```diff
@@ -0,0 +1,70 @@
+name: Air quality monitor 6 in 1
+products:
+  - id: ugjbnqj2mffaexp5
+    name: Air Housekeeper
+primary_entity:
+  entity: sensor
+  name: PM2.5
+  class: volatile_organic_compounds
+  icon: "mdi:factory"
```

Comment:
Also should not need an icon, as the class should come with an appropriate icon as well.